<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workoutWindow</class>
 <widget class="QDialog" name="workoutWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Workouts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Workout</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>32</y>
     <width>344</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_date">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date (YYMMDD):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_date"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_workout">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Workout:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_goBack">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetShowWorkouts">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_WorkoutDescription">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelShowAllWorkouts">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>All workouts:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemove">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Workout</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRemove">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add date to remove (YYMMDD):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRemove">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemoveSave">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>pushButton_2</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>label_workout</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>label_date</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
